src/OpenTelemetry.Api/.publicApi/net452/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,327 @@
+OpenTelemetry.Context.Propagation.CompositePropagator.Extract<T>(OpenTelemetry.Context.Propagation.PropagationContext context, T carrier, System.Func<T, string, System.Collections.Generic.IEnumerable<string>> getter) -> OpenTelemetry.Context.Propagation.PropagationContext
+OpenTelemetry.Context.Propagation.CompositePropagator.Fields.get -> System.Collections.Generic.ISet<string>
+OpenTelemetry.Context.Propagation.CompositePropagator.Inject<T>(OpenTelemetry.Context.Propagation.PropagationContext context, T carrier, System.Action<T, string, string> setter) -> void
+OpenTelemetry.Context.Propagation.IPropagator
+OpenTelemetry.Context.Propagation.IPropagator.Extract<T>(OpenTelemetry.Context.Propagation.PropagationContext context, T carrier, System.Func<T, string, System.Collections.Generic.IEnumerable<string>> getter) -> OpenTelemetry.Context.Propagation.PropagationContext
+OpenTelemetry.Context.Propagation.IPropagator.Fields.get -> System.Collections.Generic.ISet<string>
+OpenTelemetry.Context.Propagation.IPropagator.Inject<T>(OpenTelemetry.Context.Propagation.PropagationContext context, T carrier, System.Action<T, string, string> setter) -> void
+OpenTelemetry.Context.Propagation.PropagationContext
+OpenTelemetry.Context.Propagation.PropagationContext.ActivityContext.get -> System.Diagnostics.ActivityContext
+OpenTelemetry.Context.Propagation.PropagationContext.Baggage.get -> OpenTelemetry.Baggage
+OpenTelemetry.Context.Propagation.PropagationContext.Equals(OpenTelemetry.Context.Propagation.PropagationContext value) -> bool
+OpenTelemetry.Context.Propagation.PropagationContext.PropagationContext(System.Diagnostics.ActivityContext activityContext, OpenTelemetry.Baggage baggage) -> void
+OpenTelemetry.Context.Propagation.TextMapPropagator
+OpenTelemetry.Context.Propagation.TextMapPropagator.Extract<T>(OpenTelemetry.Context.Propagation.PropagationContext context, T carrier, System.Func<T, string, System.Collections.Generic.IEnumerable<string>> getter) -> OpenTelemetry.Context.Propagation.PropagationContext
+OpenTelemetry.Context.Propagation.TextMapPropagator.Fields.get -> System.Collections.Generic.ISet<string>
+OpenTelemetry.Context.Propagation.TextMapPropagator.Inject<T>(OpenTelemetry.Context.Propagation.PropagationContext context, T carrier, System.Action<T, string, string> setter) -> void
+OpenTelemetry.Context.Propagation.TextMapPropagator.TextMapPropagator() -> void
+OpenTelemetry.Context.RemotingRuntimeContextSlot<T>
+OpenTelemetry.Context.RemotingRuntimeContextSlot<T>.RemotingRuntimeContextSlot(string name) -> void
+OpenTelemetry.Context.RuntimeContext
+OpenTelemetry.Context.RuntimeContextSlot<T>
+OpenTelemetry.Context.RuntimeContextSlot<T>.Dispose() -> void
+OpenTelemetry.Context.RuntimeContextSlot<T>.Name.get -> string
+OpenTelemetry.Context.RuntimeContextSlot<T>.RuntimeContextSlot(string name) -> void
+OpenTelemetry.Context.ThreadLocalRuntimeContextSlot<T>
```

Comment:
same comment here

Reply:
is it something that we have to keep public?

Reply:
Hi @SergeyKanzhelev , i dont know this API either. @cijothomas , can u tell me if we can or can not remove it?

Reply:
Same as above, we need this public.